src/scripts/directives.js

```diff
@@ -307,7 +307,9 @@
                     minuteMin: '@',
                     hourStep: '@',
                     minuteStep: '@',
-                    useDuration: '@'
+                    useDuration: '@',
+                    trailHours: '@',
+                    trailMinutes: '@'
```

Comment:
If you make these `=` instead of `@`, `true` will come through as `true` instead of `'true'`, so you don't have to `JSON.parse` them. Also you can bind to angular expressions that resolve to a boolean value.